AppCenterDistribute/AppCenterDistribute/Internals/DataMigration/MSKeychainUtil+DistributeMigration.m

```diff
@@ -0,0 +1,21 @@
+#import "MSDistributeInternal.h"
+#import "MSKeychainUtil+DistributeMigration.h"
+#import "MSKeychainUtilPrivate.h"
+#import "MSUtility.h"
+
+@implementation MSKeychainUtil (DistributeMigration)
+
++ (void)migrateDistributeData {
+
+  // Migrate Mobile Center update token.
+  NSString *mcServiceName =
+      [NSString stringWithFormat:@"%@.%@", @"MobileCenter", [MS_APP_MAIN_BUNDLE bundleIdentifier]];
+  NSString *mcUpdateToken = [MSKeychainUtil stringForKey:kMSUpdateTokenKey withServiceName:mcServiceName];
+  NSString *acUpdateToken = [MSKeychainUtil stringForKey:kMSUpdateTokenKey];
+  if (!acUpdateToken && mcUpdateToken) {
```

Comment:
If both tokens exist, probably we want to delete the one for mobile center.

Reply:
Will do